### DISCUSSION TITLE
Failed rebase, error AD204/62D36/al.

### DISCUSSION THREAD

**Discussion by ffaf1** [Mergifyio/mergify#5097] [Q&A]
Hello, we are having a bit of a problem in rebasing via Mergifyio.

```
❌ Base branch update has failed

Mikolaj token is invalid, make sure Mikolaj can still log in on the [Mergify dashboard](https://dashboard.mergify.com/).
err-code: 62D36
```

([link](https://github.com/haskell/cabal/pull/9180#issuecomment-1695354640))

If you check the thread, you will note how the error code chances at each failure. I searched a bit but came out empty handed, how to get this unstuck?

**Comment by jd** [comment]
@ffaf1 Stupid question but did you make sure @Mikolaj can still log in the Mergify dashboard?

Edited: this is what GitHub replies when pushing the branch::

```
Pushing to https://github.com/fendor/cabal
remote: Permission to fendor/cabal.git denied to Mikolaj.
fatal: unable to access 'https://github.com/fendor/cabal/': The requested URL returned error: 403
```

So the problem is likely that the PR author did not allow the repo maintainer to edit its branch. We need to improve reporting for this.

**Comment by ffaf1** [comment]
Thanks, want me to file a bug?

**Reply by jd** [reply]
I just did on our side :)
